### ISSUE TITLE
Output counterexamples in the TLC format

### ISSUE THREAD

**Issue by konnov** [konnov/apalache#45]

**Comment by konnov** [comment]
The users would be able to import the counterexamples produced by Apalache in TLA+ Toolbox.

**Comment by andrey-kuprianov** [comment]
The issue is preliminary closed with the commit https://github.com/konnov/apalache/commit/3e67ca4403861c187ff0502639d8ef1cb65264fb

Further work depends on #101, and a separate issue will be created for that
